extensions/arc/notebooks/arc/_config.yml

```diff
@@ -0,0 +1,2 @@
+title: Azure Arc
+description: A collection of notebooks to support Azure Arc.
```

Comment:
Should we include the string "Data Services" anywhere, since this isn't for Azure Arc all up?

Reply:
That's a good idea.  I'll do it in other places too.